performance-tests/perf-tests.sh

```diff
@@ -59,6 +72,53 @@ warmUp() {
   consumerPerf "$1" "$2" "${WARM_UP_NUM_RECORDS_PRE_TEST}" /dev/null > /dev/null
 }
 
+setupAsyncProfilerKroxy() {
+  sudo sysctl kernel.perf_event_paranoid=1
+  sudo sysctl kernel.kptr_restrict=0
```

Comment:
How about?
```suggestion

ensureSysCtlValue() {
  local keyName=$1
  local desiredValue=$2

  if [ "$(sysctl -n ${keyName})" != ${desiredValue} ] ;
  then
    echo -e "${YELLOW}setting ${keyName} to ${desiredValue}${NOCOLOR}"
    sudo sysctl ${keyName}=${desiredValue}
  fi
}

setupAsyncProfilerKroxy() {
  ensureSysCtlValue kernel.perf_event_paranoid 1
  ensureSysCtlValue kernel.kptr_restrict 0

```

might be nice to also make these settings optional.